lldb/source/Plugins/TypeSystem/Swift/TypeSystemSwiftTypeRef.h

```diff
@@ -514,12 +522,6 @@ class TypeSystemSwiftTypeRef : public TypeSystemSwift {
   CompilerType LookupClangForwardType(llvm::StringRef name, 
                   llvm::ArrayRef<CompilerContext> decl_context);
 
-  /// Recursively resolves all type aliases.
-  swift::Demangle::NodePointer
```

Comment:
ResolveAllTypeAliases is gone from the header but not implementation?

Reply:
It never existed!